Api/Health Checks.md

```diff
@@ -0,0 +1,125 @@
+ASP.NET Core offers the built-in Health Checks feature for monitoring and reporting the health and availability of our application infrastructure components. Health checks provide a quick and automated way to verify that our application’s dependencies and components are running smoothly. They help us proactively identify issues within our application, allowing us to address them before they impact users. Regularly verifying the health of our application components can ensure a more reliable and resilient system.
+
+## Basic Health Check
+
+For many apps, a basic health probe configuration that reports the app's availability to process requests (liveness) is sufficient to discover the app's status.
+
+The basic health probe is very simple to implement. All we have to do is register health check services and configure the Health Checks Middleware to respond at a URL endpoint with a health response in `Program.cs`: 
+
+```c#
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddHealthChecks();
+
+var app = builder.Build();
+
+app.MapHealthChecks("/api/health");
+
+app.Run();
+```
+
+After adding these, we can access our application on `"/api/health"` and the application will respond with one of the three distinct `HealthStatus` values:
+* `HealthStatus.Healthy`
+* `HealthStatus.Degraded`
+* `HealthStatus.Unhealthy`
+
+## Custom Health Checks
+
+However, we often need to do more than just confirm the app can respond at the health endpoint URL to make sure it is healthy, such as checking the database connection or a third-party service connection.
+
+In the [AspNetCore.Diagnostics.HealthChecks](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks) repository we can find a wide collection of health check packages for frequently used services and libraries, some of which are:
```

Comment:
I think we'll need to add an empty line after this, the parser used to create a Handbook page has issues with special blocks (e.g. bullet list) if it's not surrounded by empty lines. (I know that this is not a part of the MD standard, but it is what it is 🤷)